app/controllers/declaration/AdditionalProcedureCodesController.scala

```diff
@@ -115,6 +125,14 @@ class AdditionalProcedureCodesController @Inject()(
       case None => (None, emptyProcedureCodesData)
     }
 
+  private def getHintText()(implicit request: JourneyRequest[AnyContent]): Option[String] =
```

Comment:
Any particular reason to not place this in a small helper under `views/helpers`? IMO we should try to have any logic concerning the pages/templates in that folder and not in controllers. But open to discuss the idea.

Reply:
No particular reason, just an oversight on my part, thanks for pointing this out.